starwars-application/src/Home.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+class Home extends Component{
+	
+    constructor(props) {
+		
+        super(props);
+
+        this.state = {
+          data: {},
+          isLoading: true,
+          hasError: false
+        }
+    }
+	
+	componentDidMount() {
+		
```

Comment:
Could also consider putting the fetch in App and passing data via props so characters only need to load once, or copying to local storage.